src/components/Notifications/form/event.ts

```diff
@@ -457,9 +454,9 @@ export const getEventSchema = (
       items: {
         "ui:ObjectFieldTemplate": RegexSelectorRow,
         regexSelect: {
-          "ui:widget": AntdSelect,
           "ui:data-cy": "regex-select",
-          "ui:disabledEnums": regexEnumsToDisable,
+          "ui:enumDisabled": regexEnumsToDisable,
+          "ui:allowDeselect": false,
         },
         regexInput: {
```

Comment:
On the notifications tab, we need to disable deselecting for these fields:

<img width="425" alt="Screen Shot 2022-12-14 at 4 46 54 PM" src="https://user-images.githubusercontent.com/47064971/207722083-88a757c0-2a9b-4f46-bcdc-c7595fb97a38.png">

<img width="434" alt="Screen Shot 2022-12-14 at 4 46 22 PM" src="https://user-images.githubusercontent.com/47064971/207722116-dd4acb7e-e16c-4774-8e8f-105dc0105f52.png">

Reply:
Great catch!